### PR TITLE
MAUI - Provide UG for cell selection

### PR DESCRIPTION
**Pull request by AbhishekSF4201**
## Description
- Have added the UG of cell selection

### CODE REVIEW

File: MAUI/DataGrid/selection.md [hunk 1]
```diff
@@ -57,6 +99,26 @@ The `SfDataGrid` provides [SelectedIndex](https://help.syncfusion.com/cr/maui/Sy
 
 `SfDataGrid.SelectedRows`: Provides all the selected records of the selected items when multiple selection is enabled.
 
+### Getting selected cells.
+
+The `SfDataGrid` provides[`SfDataGrid.GetSelectedCells()`] and [`SfDataGrid.GetSelectedCellInfo()`].
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/selection.md:104]: The `SfDataGrid` provides [SfDataGrid.GetSelectedCells()]() and [SfDataGrid.GetSelectedCellInfo()]() methods to get details of the selected cells when the selection unit is `Cell` or `Any`.

File: MAUI/DataGrid/selection.md [hunk 2]
```diff
@@ -57,6 +99,26 @@ The `SfDataGrid` provides [SelectedIndex](https://help.syncfusion.com/cr/maui/Sy
...
+
+`SfDataGrid.GetSelectedCells()` : Provides all the selected cell, when multiple cells are selected.
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/selection.md:106]: Provides details of all selected cells when multiple cells are selected

File: MAUI/DataGrid/selection.md [hunk 3]
```diff
@@ -57,6 +99,26 @@ The `SfDataGrid` provides [SelectedIndex](https://help.syncfusion.com/cr/maui/Sy
...
+
+{% tabs %}
+{% highlight c# %}
+var selectedCells = this.dataGrid.GetSelectedCells();
+{% endhighlight %}
+{% endtabs %}
+
+`SfDataGrid.GetSelectedCellInfo()` : Provides the selected cell at the specified row and column index.
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/selection.md:114]: Provides details of the cell at the specified row and column index.

**Review by ashok-kuvaraja — CHANGES_REQUESTED** [review]

File: MAUI/DataGrid/selection.md (hunk 1 above)
Comment (AbhishekSF4201) [reply MAUI/DataGrid/selection.md]: Addressed the changes

File: MAUI/DataGrid/selection.md (hunk 2 above)
Comment (AbhishekSF4201) [reply MAUI/DataGrid/selection.md]: Addressed the changes

File: MAUI/DataGrid/selection.md (hunk 3 above)
Comment (AbhishekSF4201) [reply MAUI/DataGrid/selection.md]: Addressed the changes

**Review by ashok-kuvaraja — APPROVED** [review]
The changes are fine

**Pull request merged**